plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsVmExecutorService.java

```diff
@@ -53,8 +56,30 @@ public void run() {
      * That makes it worth reporting.
      */
     private void reportProblem(Throwable t) {
+        if (isShutdown()) {
```

Comment:
Or maybe better to check this instead:
```suggestion
        if (cpsThreadGroup.getExecution().isComplete()) {
```

Reply:
Not sure about the micro specifics between the two, but using `getExecution` at least looks more readable to me.

Reply:
See [b9080e8](https://github.com/jenkinsci/workflow-cps-plugin/pull/780/commits/b9080e88143ad3eb6cdb6e57ad5fdd91abab9ce4).